Assets/Scripts/Buddy/BackBuddy.cs

```diff
@@ -44,6 +46,6 @@ public void PlayEvent( string eventName )
 	{
 		// Second param here is the animationLayer to play an event on
 		// 0 is the default layer, 1 is the face layer
-		animator.Play( eventName, 0 );
+		//animator.Play( eventName, 0 );
```

Comment:
This was commented out to fix an error that was happening, the error it self should be fixed instead.